Makefile

```
@@ -0,0 +1,79 @@
+
+# Put it first so that "make" without argument is like "make help"
+help:
+	@echo "Please use 'make <target>' where <target> is one of"
+	@echo "  html       to make standalone HTML files"
+	@echo "  clean      to remove all build files"
+	@echo "  doxygen    to run Doxygen and generate XML"
+	@echo "  livehtml   to autobuild and serve docs locally"
+	@echo "  test       to test the build locally"
+	@echo "  view       to open the docs in browser"
+
+.PHONY: help Makefile
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)